chartpress.py

```diff
@@ -1026,10 +1117,16 @@ def main(args=None):
     #   - push chart (--publish-chart, --extra-message)
     for chart in config["charts"]:
         forced_version = None
+        # TODO: maybe warn and switch default in the future?
+        use_chart_version = chart.get("useChartVersion", False)
+
         if args.tag:
+            # tag specified, use that version
             forced_version = args.tag
-        elif args.reset:
-            forced_version = chart.get("resetVersion", "0.0.1-set.by.chartpress")
+        elif args.reset and not use_chart_version:
+            # resetting, get version from chartpress.yaml,
+            # ignoring current version in Chart.yaml
+            forced_version = chart.get("resetVersion", "0.0.1-0.dev")
```

Comment:
This would be another small breaking change, so I'd prefer to let this remain as it was to not complicate things.

```suggestion
            forced_version = chart.get("resetVersion", "0.0.1-set.by.chartpress")
```